app/helpers/admin/site_maps_helper.rb

```diff
@@ -7,7 +7,7 @@ module Admin::SiteMapsHelper
   #
   # @!visibility private
   def sitemap_pages
-    @sitemap_pages ||= proper_pages.only(:id, :name, :page_id, :path).group_by(&:page_id)
+    @sitemap_pages ||= proper_pages.only(:id, :name, :page_id, :path, :position, :role).group_by(&:page_id)
```

Comment:
As with https://github.com/withassociates/slices/pull/116/files#r50687277, I don't understand why this became necessary.